### DISCUSSION TITLE
Add about page to node

### DISCUSSION THREAD

**Discussion by nickdebesten** [pelican-dev/panel#219] [Ideas]
As the title says, add an about page to the nodes like in pterodactyl. like this:

Information:

* [x] OS (name\version):
* [x] Daemon\wings version: 
* [x] CPU Threads: (number)
* [x] (AVG) CPU Load (a thread?): 
* [x] (AVG) Memory Usage
* [x] Memory Allocated
* [x] Disk Space Used
* [x] Disk Space Allocated

So you can see at a glance how much is currently being used and how much is allocated. Let me know if you have questions :)

**Comment by lancepioch** [comment]
This is what we currently have:

```php
  [
    "memory_total" => 8221421568,
    "memory_used" => 626753536,
    "swap_total" => 1073737728,
    "swap_used" => 2908160,
    "load_average1" => 0.27,
    "load_average5" => 0.33,
    "load_average15" => 0.31,
    "cpu_percent" => 0.054478060386411,
    "disk_total" => 62671097856,
    "disk_used" => 19022028800,
  ]

  [
    "architecture" => "amd64",
    "cpu_count" => 12,
    "kernel_version" => "6.6.26-linuxkit",
    "os" => "linux",
    "version" => "",
  ]

**Comment by rmartinoscar** [comment]
Closed with [PR #459](https://github.com/pelican-dev/panel/pull/459)
